packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/cancel-resiliation/translations/Messages_es_ES.json

```diff
@@ -1,3 +0,0 @@
-{
-  "autorenew_service_cancel_resiliation_success": "Se ha cancelado la baja del servicio: {{ service }}."
```

Comment:
Not used ?

Reply:
i believe it's not, it's used here : https://github.com/ovh/manager/tree/c645824395828ca08b2352dace31dce34926344a/packages/manager/modules/billing/src/components/cancel-resiliation
(you can see the translation exists on the module)